app/src/main/java/chat/rocket/android/userdetails/ui/UserDetailsFragment.kt

```diff
@@ -96,7 +103,7 @@ class UserDetailsFragment : Fragment(), UserDetailsView {
         ).into(image_blur)
 
         requestBuilder.apply(RequestOptions.bitmapTransform(RoundedCorners(14)))
-            .into(image_avatar)
+            .into(user_image_avatar)
```

Comment:
Please make sure to follow our coding style before changing the things! Wha's the reason to change the id of the imageView here?